src/components/map-components/google-map.js

```diff
@@ -96,49 +114,51 @@ export default class GoogleMap extends Component {
     console.warn('User declined to use browser location');
   }
 
-  onZoomChanged(miles) {
-    // todo: major work here bro
-    // https://stackoverflow.com/questions/52411378/google-maps-api-calculate-zoom-based-of-miles
-  }
+  /******************************************************************
```

Comment:
thank you for these! they always help people make heads or tales of the situation lol